lib/fastlane/plugin/wpmreleasetoolkit/versioning/calculators/build_code_calculator.rb

```diff
@@ -1,8 +1,8 @@
 # The `BuildCodeCalculator` class is responsible for performing calculations on build codes.
-require_relative '../models/build_code'
+require_relative '../../models/build_code'
 
-module Fastlane
-  module Calculators
+module WPMReleaseToolkit
+  module Versioning
```

Comment:
I just realized that we already have a module dedicated to our plugin, that is called `Fastlane::Wpmreleasetoolkit`, and is already used in `…/plugin/wpmreleasetoolkit`, `…/plugin/wpmreleasetoolkit/version.rb` and `…/helper/user_agent.rb`.

So for the sake of consistency we should probably stick with that namespace everywhere?

Reply:
Ah good point. I'll switch these classes over to that namespace.

Reply:
Updated in https://github.com/wordpress-mobile/release-toolkit/commit/37bf610e8a2771aea1097c7aab173b9512c93477